### PR TITLE
Scala-Wechaty

### PR DESCRIPTION
**Pull request by shraddhavp**
this PR links to #890 and this PR contains the updated section of Scala-Wechaty.

### CODE REVIEW

File: docusaurus/docs/polyglot/scala/overview.md [hunk 1]
```diff
@@ -6,28 +6,47 @@ sidebar_label: Scala
 
 [![Scala Wechaty](https://img.shields.io/badge/Wechaty-Scala-890)](https://github.com/wechaty/scala-wechaty)
 
-- GitHub Repo <https://github.com/wechaty/scala-wechaty>
-- Getting Started Template Repo <https://github.com/wechaty/scala-wechaty-getting-started>
+Scala Wechaty is a Scala programming client derived from the Wechaty ecosystem.Scala's static types help avoid bugs in complex applications,hence Scala wechaty is very easy to use,implement and is also very efficient.You can find more information on Scala-Wechaty [here](https://github.com/wechaty/scala-wechaty-getting-started).The [General Architecture Diagram](https://wechaty.js.org/docs/polyglot/diy/) illustrates on how the Scala-Wechaty can be implemented on the already existing TypeScript Wechaty ecosystem.
```
Comment (iamrajiv) [inline docusaurus/docs/polyglot/scala/overview.md]:
```suggestion
Scala Wechaty is a Scala programming client derived from the Wechaty ecosystem. Scala's static types help avoid bugs in complex applications, hence Scala wechaty is very easy to use, implement and is also very efficient. You can find more information on Scala-Wechaty [here](https://github.com/wechaty/scala-wechaty-getting-started). The [General Architecture Diagram](https://wechaty.js.org/docs/polyglot/diy/) illustrates how the Scala-Wechaty can be implemented on the already existing TypeScript Wechaty ecosystem.
```

File: docusaurus/docs/polyglot/scala/overview.md [hunk 2]
```diff
@@ -6,28 +6,47 @@ sidebar_label: Scala
...
+export WECHATY_PUPPET=wechaty-puppet-hostie
+export WECHATY_PUPPET_HOSTIE_TOKEN=your_token_at_here
+make bot # or
+scala examples/ding-dong-bot.scala 
 ```
 
-## Blogs
+## Translation of TypeScript to Scala
+
+There are a hundred (100) lines class named `Image` in charge of downloading the WeChat image to different sizes.Below are the links to Image class for both TypeScript and Scala for better understanding the code and the translation.
```
Comment (iamrajiv) [inline docusaurus/docs/polyglot/scala/overview.md]:
```suggestion
There is a hundred (100) lines class named `Image` in charge of downloading the WeChat image to different sizes. Below are the links to the Image class for both TypeScript and Scala for a better understanding of the code and the translation.
```

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: docusaurus/docs/polyglot/scala/overview.md [hunk 3]
```diff
@@ -6,28 +6,48 @@ sidebar_label: Scala
 
 [![Scala Wechaty](https://img.shields.io/badge/Wechaty-Scala-890)](https://github.com/wechaty/scala-wechaty)
 
-- GitHub Repo <https://github.com/wechaty/scala-wechaty>
-- Getting Started Template Repo <https://github.com/wechaty/scala-wechaty-getting-started>
+Scala Wechaty is a Scala programming client derived from the Wechaty ecosystem.Scala's static types help avoid bugs in complex applications,hence Scala wechaty is very easy to use,implement and is also very efficient.You can find more information on Scala-Wechaty [here](https://github.com/wechaty/scala-wechaty-getting-started).The [General Architecture Diagram](https://wechaty.js.org/docs/polyglot/diy/) illustrates on how the Scala-Wechaty can be implemented on the already existing TypeScript Wechaty ecosystem.
+
+## Features of Scala-Wechaty
+
+* Stable interaction with back-end GRPC, continuous and stable operation for many days.
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/polyglot/scala/overview.md]: after reading this it seems like, Scala-Wechaty becomes unstable after some period of time 🤔 

Also, a nitpicking suggestion is to change GRPC to gRPC 😅

**Comment by Rohitesh-Kumar-Jain** [comment]
@shraddhavp can we expect a lesser number of typos and punctuation errors in the upcoming PRs : )
Thanks for @iamrajiv for spotting them

**Comment by shraddhavp** [comment]
> @shraddhavp can we expect a lesser number of typos and punctuation errors in the upcoming PRs : )
> Thanks for @iamrajiv for spotting them

yes sure I am trying it this time  typos were less but punctuation was a bit problem but thanks to you @Rohitesh-Kumar-Jain  @proudofsimin and @iamrajiv  for correcting me next PR's you will not see typos or punctuation  errors much

**Comment by Rohitesh-Kumar-Jain** [comment]
> next PR's you will not see typos or punctuation errors much

It's okay if there are typos or punctuation errors, we are here to correct them.

But proofread yourself once before pushing.

**Review by proudofsimin — APPROVED** [review]
Thank you.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by Soumi7 — APPROVED** [review]

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by Rohitesh-Kumar-Jain** [comment]
@iamrajiv

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**